src/components/search-bar.component.ios.js

```diff
@@ -12,6 +12,13 @@ export class SearchBar extends Component {
     onSearchButtonPress: PropTypes.func.isRequired,
   };
 
+  static defaultProps = {
```

Comment:
Cool, I am not familiar with flow. Looks better.
Fixed in search-bar ios & android.